### DISCUSSION TITLE
payment form template

### DISCUSSION THREAD

**Discussion by sammy429b** [mdbootstrap/TW-Elements#1954] [Share ideas & request features]
We can add payment form in form templates it will benifit.
![paymenr form](https://github.com/mdbootstrap/Tailwind-Elements/assets/97588853/f39d3a8c-812f-494c-a858-df47f8a0723a)

**Comment by ascensus-mdb** [comment]
Thank you for your contribution. You can add this example to our snippets collection by using playground:

https://tailwind-elements.com/snippets/

Just click "Create snippet" and after adding your code publish it, so everyone can see it and use it.
